src/Antd_Avatar.re

```diff
@@ -0,0 +1,53 @@
+open Antd_Helpers;
+
+module IconName = Antd_IconName;
+
+[@bs.module] external reactClass : ReasonReact.reactClass = "antd/lib/avatar";
+
+[%bs.raw {|require("antd/lib/avatar/style")|}];
+
+[@bs.deriving jsConverter]
+type avatarShape = [ | `circle | `square];
+
+[@bs.deriving jsConverter]
+type avatarSize = [ | `small | `default | `large];
+
+[@bs.obj]
+external makeProps :
+(
+    ~shape: string=?,
+    ~size: string=?,
+    ~src: string=?,
+    ~icon: IconName.t=?,
+    ~style: ReactDOMRe.Style.t=?,
+    ~prefixCls: string=?,
+    ~className: string=?,
```

Comment:
I have id, className and style props in all components because they are common html properties. See https://github.com/thangngoc89/bs-ant-design/blob/master/src/Antd_Button.re#L50-L52

Reply:
k cool il add those